network_params.yaml

```diff
@@ -2,11 +2,12 @@ participants:
   - el_type: geth
     cl_type: lighthouse
     count: 2
-    validator_count: 32
+    validator_count: 1
   - el_type: geth
     cl_type: lambda
     cl_image: lambda_ethereum_consensus:latest
     use_separate_vc: false
     count: 1
-    validator_count: 32
-    cl_max_mem: 4096
\ No newline at end of file
+    validator_count: 63
```

Comment:
np: do we want to have 1 in every lighthouse node and 63 (62 would be sufficient) in ours as the new default config?